doc/formato-artigo.md

```diff
@@ -94,7 +94,6 @@ formato que um parágrafo.
 
 Umha imagem tem as seguintes propriedades:
 
-- nome: nom se mostra ao público, é por facilidade para encontrá-la nós.
```

Comment:
Esta mudança si é para importar a este repositório.  Mais tarde hei mandar um pull request máis limpo.